custom/shared/src/main/java/com/solarwinds/opentelemetry/extensions/DelegatingMetricExporter.java

```diff
@@ -33,7 +36,19 @@ public DelegatingMetricExporter(MetricExporter delegate) {
 
   @Override
   public CompletableResultCode export(@NonNull Collection<MetricData> metrics) {
-    return delegate.export(metrics);
+    if (ConfigManager.getConfigOptional(ConfigProperty.AGENT_EXPORT_METRICS_ENABLED, true)) {
+      return delegate.export(metrics);
+    }
+
+    return delegate.export(
+        metrics.stream()
+            .filter(
+                metricData ->
+                    MeterProvider.samplingMeterScopeName.equals(
+                            metricData.getInstrumentationScopeInfo().getName())
+                        || MeterProvider.requestMeterScopeName.equals(
+                            metricData.getInstrumentationScopeInfo().getName()))
```

Comment:
Tiny little nitpick but `metricData.getInstrumentationScopeInfo().getName()` twice is a little long